src/mame/drivers/nes_vt.cpp

```diff
@@ -99,6 +99,11 @@ class nes_vt_state : public nes_base_state
 	DECLARE_READ8_MEMBER(spr_r);
 	DECLARE_READ8_MEMBER(chr_r);
 
+	/* expansion nametable */
+	std::vector<uint8_t> m_ntram;
```

Comment:
If you're not going to be resizing this on-the-fly you should use `std::unique_ptr<uint8_t []>` or an array in the object.  Also, the member probably shouldn't be public.